bin/offline-helm.sh

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env bash
 
 set -euo pipefail
+set -x
```

Comment:
Is this a debugging left-over? Or, should this be upstreamed?

Reply:
this is needed for the debugging purpose of the failing CI